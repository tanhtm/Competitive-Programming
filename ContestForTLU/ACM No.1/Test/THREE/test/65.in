991459610000000000
