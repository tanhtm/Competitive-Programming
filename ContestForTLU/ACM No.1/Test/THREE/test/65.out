0 5 24 34 35 37 
2 4 6 7 11 17 18 21 22 26 28 29 32 33 36 38 
